README.md

```diff
@@ -1,57 +1,35 @@
-# p-report-template
+# p-report
+[![Build Status](https://travis-ci.org/jsk-report-template/p-report.svg?branch=master)](https://travis-ci.org/jsk-report-template/p-report)
 
 Latex template for p-report
 
-[![Build Status](https://travis-ci.org/furushchev/p-report-template.svg)](https://travis-ci.org/furushchev/p-report-template)
-
 ### 1. Prerequisities
 
 ```bash
-# only for ubuntu 12.04
```

Comment:
Currently, Ubuntu Precise is supported: https://github.com/jsk-report-template/si-template/pull/4#issue-421101801
Do you know this current policy?
If you decide to remove Precise support though you know the current policy, it's ok (I'm sad, though...).
I'll fix other repositories under jsk-report-template to follow the new policy.

Reply:
hmm, from the viewpoint of maintenance, we don't have presice test, so we cannot support it anymore, I think.

Reply:
OK, thanks